### ISSUE TITLE
Criar depósitos

### ISSUE THREAD

**Issue by victorperin** [gserrano/nerraw#10]
Olá, parabéns pela iniciativa.

Existe um [endpoint](https://blinktrade.com/docs/#deposits)  para criar depósitos.

Podemos usa-lo para simplificar o processo de depósito, o que acha? :)

**Comment by gserrano** [comment]
Acho que pode ser uma ótima idéia @victorperin! Vi que há endpoint parta saques também.

Como você acha que isso deveria funcionar? Um item no menu para criar um novo depósito e um para saque?

**Comment by victorperin** [comment]
Sim, mas pelo que entendi, vai haver alguns passos. Por exemplo:

```
- Seleciono criar deposito
- Seleciono BTC ou BRL
  - Se for BTC
    - mostra a carteira que tenho que depositar
    - (opcional) fica de olho no endereço até que haja um pagamento
    - (opcional) aguarda as confirmações necessárias e avisa quando terminar
  - Se for BRL
    - Seleciono o banco
    - Aparecem informações de depósito e o numero de confirmação
```

Isso é só um esboço do que pode se tornar (pelo menos o que eu pensei ontem haha), qualquer idéia/modificação nesse fluxo é bem vinda.
